.github/workflows/python-package.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CRDS_SERVER_URL: "https://jwst-crds.stsci.edu"
```

Comment:
Need to make sure downloads from server won't hit limits to the point you get charged real money for CI; if it does, work with management to see if charges are acceptable. If not, might want to consider running data intensive CI on internal Jenkins instance.

Also not sure if the server would start to blacklist you if you hit it too much. If it does, you need to work with ITSD to see if Actions IPs can be whitelisted.